app/plugins/statsd.md

```diff
@@ -54,7 +54,7 @@ parameter                     | default | description
 
 ## Metrics
 
-Plugin currently logs four metrics to the StatsD server: `request_count`, `request_size`, `response_size`, `status_count` and `latency`.
+Plugin currently logs four metrics to the StatsD server: `request_count`, `request_size`, `response_size`, `latency`. `status_count`, `unique_users`, `request_per_user`, and `upstream_latency`.
```

Comment:
@coopr @mgiorgio  Yes next major will have 4 new metrics and more flexible configuration. Doc will also be updated accordingly. I am closing this PR as it will covered by another PR.